test/models/request/CashTransactionsSearchRequestSpec.scala

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.request
+
+import play.api.libs.json.{JsSuccess, Json}
+import utils.SpecBase
+
+class CashTransactionsSearchRequestSpec extends SpecBase {
+
+  "CashAccountTransactionSearchRequestDetailsSpec" must {
+
+    "populate correctly" in new Setup {
```

Comment:
Please remove this test as its only populating the class and of no use. Its inherent functionality